ingest-app/src/cmr/ingest/api/ingest/collections.clj

```diff
@@ -0,0 +1,69 @@
+(ns cmr.ingest.api.ingest.collections
```

Comment:
Nit: Not sure I like the structure of ingest/api/ingest/... Why not put all of these refactored namespaces under ingest/api? I get that they were replacing a namespace called ingest.clj, but I think all of these are fine to group together at api.